### DISCUSSION TITLE
Retain position when moving window between displays

### DISCUSSION THREAD

**Discussion by johansolve** [rxhanson/Rectangle#675] [Ideas]
- [x] This is a new bug that hasn't been documented in a closed issue or in the Readme.

When moving a window to another display, the current window position should be retained. Currently the window is centered on the new display. 
If moving to a display with a different resolution, the position should be retained relative to the closest edges of the display. 

macOS version:10.14.6
Rectangle version:0.49 (54)
Logs if applicable (In Rectangle menu, hold option, "View Logging..."):

**Comment by rxhanson** [comment]
Centering the window is intentional, as it reduces complexity and ambiguity for scenarios like different size displays and resulting window size. 

But, this is commonly requested and I see no reason to reject a good pull request with it.
